app/jobs/claim_verifier_job.rb

```diff
@@ -1,14 +1,20 @@
 class ClaimVerifierJob < ApplicationJob
   def perform(claim)
-    if claim.eligibility.respond_to?(:teacher_reference_number)
-      AutomatedChecks::ClaimVerifier.new(
-        claim:,
-        dqt_teacher_status: claim.has_dqt_record? ? Dqt::Teacher.new(claim.dqt_teacher_status) : Dqt::Client.new.teacher.find(
-          claim.eligibility.teacher_reference_number,
-          birthdate: claim.date_of_birth.to_s,
-          nino: claim.national_insurance_number
-        )
-      ).perform
+    dqt_teacher_status = if claim.policy == Policies::EarlyYearsPayments
+      nil
+    elsif claim.has_dqt_record?
+      Dqt::Teacher.new(claim.dqt_teacher_status)
+    else
+      Dqt::Client.new.teacher.find(
+        claim.eligibility.teacher_reference_number,
+        birthdate: claim.date_of_birth.to_s,
+        nino: claim.national_insurance_number
+      )
```

Comment:
this isn't great but will do for now. ideally each automated check should be independent and all called async which is not the case now